app/src/main/java/com/github/swent/echo/compose/association/AssociationMainScreen.kt

```diff
@@ -73,6 +73,8 @@ fun AssociationExpandableList(
     val topBoxHeight = 80.dp
     val paddingTextStart = 10.dp
     val paddingIconEnd = 10.dp
+    val textColor = MaterialTheme.colorScheme.onPrimaryContainer
```

Comment:
Good point! However, I'm not sure how we could do it, because we would need some constant which we would pass around multiple files, i.e. define it as some global constant. This doesn't seam very clean...